hyperjumptech/monika
# Fix undefined bug in CLI logs

# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. Fixes Bug #254

## How did you implement / how did you fix it  
1. Rewired the logging structure to use emitter
2. Rewired how logs/notifications are generated

## How to test  
1. Setup monika.json with multiple probes, preferably ones with log delay
2. Enable notifications to detect service up/down interruptions/recovery
You should not get the log displayed anymore:
`WARN: undefined`
